### DISCUSSION TITLE
Datalabel OffsetY is not working on highest bars

### DISCUSSION THREAD

**Discussion by joaofuzessy** [apexcharts/apexcharts.js#3344] [Q&A]
When working with bar charts, whenever I have grouped bars, the highest bar in the series have its dataLabel touching the bar, even if the offsetY is set, as follows. It is possible to observe that the second bar in the image is not using the appropriate Offset.
I have found a similar issue [here](https://stackoverflow.com/questions/69069841/apexchart-100-touches-the-bar-though-i-have-offsety-as-30) and also [here](https://github.com/apexcharts/apexcharts.js/issues/1102) but with no solution.

` dataLabels: {
      offsetY: -16,
    },`

![Captura de Tela 2022-08-30 às 15 48 14](https://user-images.githubusercontent.com/10121028/187518993-e3b0c59f-c75a-4e24-b056-113be554878e.png)

**Comment by marcelodearaujopedro** [comment]
same problem here
